run_neuron.py

```diff
@@ -0,0 +1,82 @@
+"""
+Thank you to Namoray of SN19 for their autoupdate implementation!
+"""
+import os
+import sys
+import subprocess
+import time
+import argparse
+
+# self heal restart interval
+RESTART_INTERVAL_HOURS = 6
+
+
+def should_update_local(local_commit, remote_commit):
+    return local_commit != remote_commit
+
+
+def run_auto_update_self_heal(neuron_type, network, auto_update, self_heal):
+    last_restart_time = time.time()
+
+    while True:
+        time.sleep(60)
```

Comment:
What was the reasoning on choosing this timeout period?

Reply:
This is the cadence for checking for new updates, I left it as the default found in sn19's implementation.